test/models/game_test.rb

```diff
@@ -1,7 +1,185 @@
 require 'test_helper'
 
 class GameTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test "number of pawns" do
+    game = Game.create
+
+    # puts Piece.where( :type => 'Pawn' ).pluck( :type, :x_position, :y_position ).inspect
+
+    num_of_pawns = Piece.where( :type => 'Pawn', :game_id => game.id ).count
+
+    assert_equal 16, num_of_pawns
+  end
+
+  test "correct pawn positions" do
+    game = Game.create
+
+    correct_positions = [[1, 2], [2, 2], [3, 2], [4, 2],
+                         [5, 2], [6, 2], [7, 2], [8, 2],
+                         [1, 7], [2, 7], [3, 7], [4, 7],
+                         [5, 7], [6, 7], [7, 7], [8, 7]]
+
+    actual_positions = game.pieces.where( :type => 'Pawn' ).pluck( :x_position, :y_position )
+
+    # result should be an empty array if the positions are correct
+    result =  correct_positions - actual_positions
```

Comment:
instead of doing this, could you?

```Ruby
assert_equal correct_positions, actual_positions
```

(not sure assert_equal can even compare arrays, just thinking it would be easier to read if it can :smile:)